sdks/java/extensions/ml/build.gradle

```diff
@@ -26,15 +28,23 @@ description = 'Apache Beam :: SDKs :: Java :: Extensions :: ML'
 dependencies {
     compile project(path: ":sdks:java:core", configuration: "shadow")
     compile project(":sdks:java:expansion-service")
-    testCompile project(path: ':sdks:java:core', configuration: 'shadowTest')
     compile 'com.google.cloud:google-cloud-video-intelligence:1.2.0'
+    compile 'com.google.cloud:google-cloud-dlp:1.1.1'
```

Comment:
Can we change this to 1.1.4? There are some known issues with 1.1.1 and 1.1.2

Reply:
Sure, thanks for this insight.